pkg/api/v1alpha1/cluster_webhook_test.go

```diff
@@ -1442,23 +1150,90 @@ func TestClusterUpdateControlPlaneTaintsAndLabelsSuccess(t *testing.T) {
 }
 
 func TestClusterUpdateControlPlaneLabelsInvalid(t *testing.T) {
-	cOld := &v1alpha1.Cluster{
+	cOld := createCluster()
+	cOld.Spec.ControlPlaneConfiguration = v1alpha1.ControlPlaneConfiguration{
+		Labels: map[string]string{
+			"test": "val1",
+		},
+	}
+	cOld.SetManagedBy("management-cluster")
+	c := cOld.DeepCopy()
+	c.Spec.ControlPlaneConfiguration.Labels["test"] = "val1/val2"
+
+	g := NewWithT(t)
+	g.Expect(c.ValidateUpdate(cOld)).NotTo(Succeed())
+}
+
+func TestClusterValidateCreateFail(t *testing.T) {
+	t.Setenv(features.FullLifecycleAPIEnvVar, "true")
+	cOld := createCluster()
+	cOld.Spec.ControlPlaneConfiguration = v1alpha1.ControlPlaneConfiguration{}
+	cOld.SetManagedBy("management-cluster")
+
+	g := NewWithT(t)
+	g.Expect(cOld.ValidateCreate()).NotTo(Succeed())
+	strings.Contains(cOld.ValidateCreate().Error(), "cluster validation failed")
```

Comment:
I believe this is not testing anything, we are not even checking the result of Contains

I would advice using gomega with an expect for error contains substring. I think there should already be examples in our codebase

Also, why call `ValidateCreate()` twice?